### PR TITLE
Few teachers and few classes for one group reorganization

### PR DESCRIPTION
**Pull request by matanel-oren**
Now the every ClassGroup has a list (by ManyToMany field) of teacher instead of only in CourseClass.
Solves the issue https://github.com/asaf-kali/coursist/issues/12

### CODE REVIEW

File: academic_helper/migrations/0011_classgroup_teachers.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    dependencies = [
+        ('academic_helper', '0010_auto_20200629_1808'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='classgroup',
+            name='teachers',
+            field=models.ManyToManyField(blank=True, null=True, to='academic_helper.Teacher'),
```
Comment (asaf-kali) [inline academic_helper/migrations/0011_classgroup_teachers.py:16]: Remove blank and null

File: academic_helper/logic/shnaton_parser.py [hunk 2]
```diff
@@ -206,11 +200,8 @@ def parse_lecturers(lecturers):
 
     for lecturer in lecturers.contents:
         # TODO: This order is not right for course 1921!
-        if lecturer.string is not None:
+        if lecturer.string is not None and lecturer.string.strip() != '':
             ret.append(lecturer.string)
-        elif lecturer.name == "br" and lecturer.previous_sibling not in ret:
-            ret.append(ret[-1])
```
Comment (asaf-kali) [inline academic_helper/logic/shnaton_parser.py:212]: Are you completely sure these lines can be removed?

Comment (matanel-oren) [reply academic_helper/logic/shnaton_parser.py:212]: Not sure. Will be reverted

File: academic_helper/migrations/0011_classgroup_teachers.py (hunk 1 above)
Comment (matanel-oren) [reply academic_helper/migrations/0011_classgroup_teachers.py]: removed in the new commit

File: academic_helper/logic/shnaton_parser.py (hunk 2 above)
Comment (matanel-oren) [reply academic_helper/logic/shnaton_parser.py]: reverted in the new commit

**Comment by asaf-kali** [comment]
Merged as #111

**Pull request closed without merging**